File: src/dto/Post.java
package dto;

public class Post {
	public int id;
	public String type;
	public String title;
	public String content;
	public String publisherName;

	public Post(int id, String type, String title, String content, String publisherName) {
		this.id = id;
		this.type = type;
		this.title = title;
		this.content = content;
		this.publisherName = publisherName;
	}
}

File: src/main/resources/static/js/class/post.js

async function postComment(lectureCode, sectionCode, postId) {
  const form = document.getElementById("form_comment");

  const content = form.comment.value;

  const url = `http://localhost:8080/api/class/${lectureCode}/${sectionCode}/post/${postId}/comment`
  const response = await fetch(url, {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({content})
  });

  if (response.ok) {
    location.reload();
  }
  else {
    const reason = (await response.json()).msg

    alert(`댓글 작성이 실패했습니다.\n사유: ${reason}`);
  }

  return false;
}